README.md

```diff
@@ -1,6 +1,6 @@
-# PREMAKE 5 (core) 
-* Linux: [![Build Status](https://travis-ci.org/premake/premake-core.svg?branch=master)](https://travis-ci.org/premake/premake-core)
-* Windows: [![Build Status](http://jenkins.tomvandijck.com/job/premake-core/badge/icon)](http://jenkins.tomvandijck.com/job/premake-core/)
+# PREMAKE 5 (core)
+* Linux  : [![Build Status](https://travis-ci.org/premake/premake-core.svg?branch=master)](https://travis-ci.org/premake/premake-core)
+* Windows: [![Build status](https://ci.appveyor.com/api/projects/status/j34shfmryvqh1c6x?svg=true)](https://ci.appveyor.com/project/tvandijck/premake-core)
```

Comment:
I think this page explains it a little, since others were confused too:

http://help.appveyor.com/discussions/questions/1154-appveyor-account-for-github-organizations